modules/ROOT/pages/as2/as2-connector-upgrade-migrate.adoc

```diff
@@ -0,0 +1,249 @@
+= AS2 Connector - Upgrade and Migrate - Mule 4
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../../assets/images/
+
+== Supported Upgrade Paths
+
+This a direct upgrade from the AS2 connector 4.x to 5.0.0.
+
+== Changes in This Release
+
+* The `Async MDN` is now handled as a separate flow. You need to add a new flow with an endpoint for the async MDN.
+
+* Users can reuse their HTTP listener configuration. Doing so means several changes for you and your trading partners:
+
+** Changing ports for an AS2-receive endpoint/port with the new HTTP listener's endpoint/port configuration
```

Comment:
```suggestion
** Changing ports for an AS2-receive endpoint or port with the new HTTP listener's endpoint or port configuration
```

Reply:
Actually it should be parens because the endpoint here is a port.

Reply:
Fixed.